open-api-spec.yaml

```diff
@@ -32,10 +32,10 @@ info:
   contact:
     email: tech@interledger.org
 servers:
-  - url: 'https://openpayments.guide'
+  - url: 'https://openpayments.guide/alice'
```

Comment:
I think we should keep `https://openpayments.guide` as URL and then add `payment-pointer` as a parameter.

Reply:
Unlike the account which was referred to by its id (as a path parameter), the payment pointer is the full base url (as demonstrated in the `servers` list)

Reply:
which is more consistent than before.
Previously `accountId` was both the uuid as a path parameter and the full account url as a field on all of the subresources.

Reply:
:thinking: Actually...
I'm not sure how to reconcile this with the [`/connections/{connectionId}` endpoint](https://github.com/interledger/open-payments/blob/main/open-api-spec.yaml#L85), which should be on the open payments server (not the payment pointer).

Reply:
Well.. yesterday we discussed potentially getting rid of `/connections`....

Reply:
I'm thinking we'd need to split [`open-api-spec.yaml`](https://github.com/interledger/open-payments/blob/main/open-api-spec.yaml) into two files:
- `payment-pointer-spec.yaml`? (everything except `/connections`)
- `connections-spec.yaml`?

Reply:
After discussing with @sabineschaller, I've reverted the servers list to not include payment pointer urls since they do not include the connections endpoints. We can consider splitting the files later if I run into issues implementing this change in Rafiki.